src/org/labkey/targetedms/chromlib/Constants.java

```diff
@@ -121,8 +121,8 @@ private Constants() {}
         CalcNeutralMass("DOUBLE NOT NULL"),
         NumMissedCleavages("INTEGER NOT NULL"),
 
-        IonFormula("VARCHAR(100)"),
-        CustomIonName("VARCHAR(100)"),
+        ChemicalFormula("VARCHAR(100)"),
+        MoleculeName("VARCHAR(100)"),
```

Comment:
I suggest to remove this and use FragmentName defined at L181 in MoleculeTransitionColumn. FragmentName column